Dockerfile

```diff
@@ -21,7 +21,7 @@
 # (MIT License)
 
 # Base image
-FROM artifactory.algol60.net/docker.io/alpine:3.13 as base
+FROM artifactory.algol60.net/docker.io/alpine:3.15 as base
```

Comment:
Don't forget to update the copyright date.

Reply:
fixed